content/tutorials/marketplace-fees.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Fees
+description: The Ocean Protocol defines various fees for creating a sustainability loop.
+---
+
+## Path to sustainability
+
+Ocean Protocol aims to be a community-driven project. Through OceanDAO, the project aims at achieving decentralized governance and sustainability loop. For the project to grow and improve, the participants need to be rewarded for their contributions. The Ocean Protocol's smart contracts include a fee mechanism to maintain the sustainability loop.
```

Comment:
Suggestions:
"aims to be" or "it is"? 
"need to be rewarded" or "are rewarded"?